apps/moderation/src/components/transparency-log.tsx

```diff
@@ -0,0 +1,293 @@
+      if (node) listItemObserver.current.observe(node);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [requesting],
+  );
+
+  const getStatusCount = async () => {
+    setRequesting(true);
+    try {
+      const response = await moderationRequest.getCount();
+      if (response) {
+        setCount(response);
+      }
+    } catch (error) {
+      logger.error('[transparency-log.tsx]: getStatusCount err %j', error.message || '');
+    } finally {
+      setRequesting(false);
+    }
+  };
+
+  const fetchModerationLog = async (offset?: string, limit?: number) => {
+    // fetch log of moderated contents
+    setRequesting(true);
+    try {
+      const response = await moderationRequest.getLog({
+        offset,
+        limit: limit || DEFAULT_LIMIT,
+      });
+      getStatusCount();
+      if (response.results) {
+        setLogItems([...logItems, ...response.results]);
+      }
+      setNextIndex(response.nextIndex);
+    } catch (error) {
+      logger.error('[transparency-log.tsx]: fetchModerationLog err %j', error.message || '');
+    } finally {
+      setRequesting(false);
+    }
+  };
+
+  const onTabClick = (value: string) => {
+    setActiveButton(buttonValues[buttonLabels.indexOf(value)]);
+  };
+
+  const handleClickArrowLeft = () => {
+    setSelected(null);
+  };
+
+  const handleClickViewItem = (contentType: string, contentID: string) => () => {
+    if (contentType === 'post') {
+      navigateToUrl(`${BASE_SOCIAL_URL}/post/${contentID}`);
+    } else if (contentType === 'reply' || contentType === 'comment') {
+      navigateToUrl(`${BASE_SOCIAL_URL}/reply/${contentID}`);
+    } else if (contentType === 'account') {
+      navigateToUrl(`${BASE_PROFILE_URL}/${contentID}`);
+    }
+  };
+
+  const handleClickAvatar = () => {
+    /* TODO */
+  };
+
+  const handleClickContactModerators = () => {
+    /* TODO */
+  };
+
+  const handleClickCard = (el: ILogItem) => () => {
+    selected?.contentID === el.contentID ? setSelected(null) : setSelected(el);
+  };
+
+  const buttonValues = !isMobile
+    ? ['All', 'Kept', 'Delisted']
+    : ['All', 'Kept', 'Delisted', 'Stats'];
+  const buttonLabels = !isMobile
+    ? [t('All'), t('Kept'), t('Delisted')]
+    : [t('All'), t('Kept'), t('Delisted'), t('Stats')];
+
+  const baseAvatarUrl = 'https://hub.textile.io/ipfs/';
+
+  const RenderDetailCard = (selected: ILogItem) => {
+    return (
+      <TransparencyLogDetailCard
+        locale="en"
+        title={t(
+          `${selected.contentType.charAt(0).toUpperCase()}${selected.contentType.substring(1)} ${
+            selected.delisted ? 'delisted by' : 'kept by'
+          }`,
+        )}
+        content={t(`${selected.explanation}`)}
+        isDelisted={selected.delisted}
+        moderator={selected.moderator.name}
+        moderatedTimestamp={selected.moderatedDate.toString()}
+        moderatorAvatarUrl={
+          baseAvatarUrl + selected.moderator.avatar || 'https://placebeard.it/360x360'
```

Comment:
Isn't the moderator just another user profile? If you are  using the useProfile hook to get the profile data for the moderator then the avatar will already contain the full image path.

Reply:
mapping through the list, since we have the avatar coming from the api response, the ipfsgateway link will work just fine